lib/rules/jsx-newline.js

```diff
@@ -45,11 +50,20 @@ module.exports = {
   create(context) {
     const jsxElementParents = new Set();
     const sourceCode = context.getSourceCode();
+
+    function isMultilined(node) {
+      return (node.loc.start.line !== node.loc.end.line);
+    }
+
     return {
       'Program:exit'() {
         jsxElementParents.forEach((parent) => {
           parent.children.forEach((element, index, elements) => {
             if (element.type === 'JSXElement' || element.type === 'JSXExpressionContainer') {
+              const configuration = context.options[0] || {};
+              const prevent = configuration.prevent || false;
+              const allowMultilines = (configuration.allowMultilines || false) && prevent;
```

Comment:
does it make sense to have prevent false, and allowMultilines true? if not, can the schema forbid that?